CovertInfiltration/Src/CovertInfiltration/Classes/X2SitRep_InfiltrationSitRepEffects.uc

```diff
@@ -140,11 +134,11 @@ static function X2SitRepEffectTemplate CreateTacticalAnalysisAbilityTemplate_CI(
 /// Encounters ///
 //////////////////
 
-static function X2SitRepEffectTemplate CreateGunneryEmplacementsEffectTemplate_CI()
+static function X2SitRepEffectTemplate CreateGunneryEmplacementsEffectTemplate()
 {
     local X2SitRepEffect_ModifyTurretCount Template;
 
-    `CREATE_X2TEMPLATE(class'X2SitRepEffect_ModifyTurretCount', Template, 'GunneryEmplacementsEffect_CI');
+    `CREATE_X2TEMPLATE(class'X2SitRepEffect_ModifyTurretCount', Template, 'GunneryEmplacementsEffect');
 
     Template.MinCount = 2;
     Template.MaxCount = 2;
```

Comment:
You force the number of turrets to be [2;2] - hence no change will ever be made. Remove the `Min`/`Max` lines and it should work